### PR TITLE
Samples fixed with nuget updates

### PR DESCRIPTION
**Pull request by EmilAlipiev**
### Description of Change ###
Samples had old nuget references and configurations. I have just fixed them that we can debug at least :)

### CODE REVIEW

File: sample/XUIComponents/UIComponents/UIComponents.Tizen.Wearable/UIComponents.Tizen.Wearable.csproj [hunk 1]
```diff
@@ -1,33 +1,34 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Tizen.NET.Sdk/1.0.9">
+
 
-  <!-- Property Group for Tizen40 Project -->
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>tizen40</TargetFramework>
+    <!--<RuntimeIdentifier>tizen-armel</RuntimeIdentifier>-->
+    <CopyLocalRuntimeTargetAssets>true</CopyLocalRuntimeTargetAssets>
```
Comment (jkpu) [inline sample/XUIComponents/UIComponents/UIComponents.Tizen.Wearable/UIComponents.Tizen.Wearable.csproj:8]: Why did you add `CopyLocalRuntimeTargetAssets` ? 
Does this sample have any library wich is included in sample app tpk?

Comment (EmilAlipiev) [reply sample/XUIComponents/UIComponents/UIComponents.Tizen.Wearable/UIComponents.Tizen.Wearable.csproj:8]: I have copied from my other project. there should be copy paste mistake i will remove it.
